### PR TITLE
Toggl docs

### PR DESCRIPTION
**Pull request by TylerMetz**
## WHAT

Updated location of toggl docs from the v8 source to the v9 source to allow easier clarity for the next changes to be made.

copilot:poem

## WHY

This was completed as I first helped update the sources of Toggl to V9 and for my current Software Enterprise class I was asked to contribute to the docs of an open source project.

### CODE REVIEW

**Comment by pipedream-component-development** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by pipedream-component-development** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:
- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/toggl/actions/get-current-time-entry/get-current-time-entry.mjs [hunk 1]
```diff
@@ -4,7 +4,7 @@ export default {
   name: "Get Current Time Entry",
   version: "0.0.3",
```
Comment (luancazarine) [inline components/toggl/actions/get-current-time-entry/get-current-time-entry.mjs:5]:
```suggestion
  version: "0.0.4",
```

File: components/toggl/actions/get-time-entries/get-time-entries.mjs [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default {
   name: "Get Time Entries",
   version: "0.0.3",
```
Comment (luancazarine) [inline components/toggl/actions/get-time-entries/get-time-entries.mjs:5]:
```suggestion
  version: "0.0.4",
```

File: components/toggl/actions/get-time-entry/get-time-entry.mjs [hunk 3]
```diff
@@ -4,7 +4,7 @@ export default {
   name: "Get Time Entry",
   version: "0.0.3",
```
Comment (luancazarine) [inline components/toggl/actions/get-time-entry/get-time-entry.mjs:5]:
```suggestion
  version: "0.0.4",
```

**Review by luancazarine — DISMISSED** [review]
Hi @TylerMetz, thanks for your contribution! I just added some comments about version.
You should also increment the version in package.json

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**